### DISCUSSION TITLE
Allow specifying a pyrightconfig path / Support monorepo configs

### DISCUSSION THREAD

**Discussion by Avasam** [microsoft/pylance-release#3426] [Enhancement]
I'd like to be able to specify a path to pyrightconfig. For example when contributing to typeshed, I'd like to configure my editor to pickup their stricter configuration file instead of the default one.

The ability to pass custom args (like `-p`) would also work for me.

Or allow pyright to override per directory in a single configuration file. (which would also solve my issue in a different manner since multiple configs won't be necessary anymore)

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
